pkg/containertools/runner.go

```diff
@@ -22,14 +22,53 @@ type CommandRunner interface {
 type ContainerCommandRunner struct {
 	logger        *logrus.Entry
 	containerTool ContainerTool
+	config        *RunnerConfig
+}
+
+type RunnerConfig struct {
+	SkipTLS bool
+}
+
+type RunnerOption func(config *RunnerConfig)
+
+func SkipTLS(skip bool) RunnerOption {
+	return func(config *RunnerConfig) {
+		config.SkipTLS = skip
+	}
+}
+
+func (r *RunnerConfig) apply(options []RunnerOption) {
+	for _, option := range options {
+		option(r)
+	}
+}
+
+func (r *ContainerCommandRunner) argsForCmd(cmd string, args ...string) []string {
+	cmdArgs := []string{cmd}
+	switch r.containerTool {
+	case PodmanTool:
```

Comment:
Don't we also need to do this for docker?

Reply:
IThe docker daemon itself maintains a list of insecure remote registries it can talk to, so we'd have to re-run the daemon with a new list of registries if we wanted to support those. Insecure registries on localhost are fine though.